low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -81,8 +82,15 @@ def get_health_info(self, rpath):
         nodes = rpath.strip().split(">")
         leaf_node, _ = self.get_node_details(nodes[-1])
         if leaf_node == "storage":
-            for fru in self.storage_frus:
-                info.update({fru: self.storage_frus[fru]()})
+            # TODO should we handle >hw>* case?
```

Comment:
> TODO should we handle >hw>* case?

ResourceMap & Health collection can happen till leaf resource, which is fru level here, we wont go granular beyond that, or we will have to maintain functions per attribute update. While consuming data, it can be filtered at required level using kvstore apis

Reply:
Okay